envs/example/defaults-2.0.yml

```diff
@@ -97,6 +97,7 @@ openstack:
   git_mirror:  https://github.com/openstack
   git_update: yes
   ubuntu_mirror: https://apt-mirror.openstack.blueboxgrid.com/trusty/ubuntu
+  upper_constraints: 'https://raw.githubusercontent.com/openstack/requirements/stable/newton/upper-constraints.txt'
```

Comment:
Will move to filemirror under new PR to support deployments without egress internet.